### PR TITLE
#72 Adding favicon link and meta tags

### PR DESCRIPTION
**Pull request by RobLoach**
#72

### CODE REVIEW

File: templates/html.html [hunk 1]
```diff
@@ -4,6 +4,15 @@
...
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  
+  <link rel="shortcut icon" href="/images/favicon.ico" type="image/vnd.microsoft.icon">
+  <link rel="apple-touch-icon" href="/images/touch-icon.png">
+  <link rel="icon" href="images/favicon.png">
+  <!--[if IE]><link rel="shortcut icon" href="/images/favicon.ico"><![endif]-->
+  <!-- or, set /favicon.ico for IE10 win -->
+  <meta name="msapplication-TileColor" content="#f00">
+  <meta name="msapplication-TileImage" content="/images/tileicon.png">
```
Comment (RobLoach) [inline templates/html.html]: What is this one?

**Comment by RobLoach** [comment]
Should we match what Boierplate has? https://github.com/h5bp/html5-boilerplate/blob/master/dist/index.html#L1

File: templates/html.html (hunk 1 above)
Comment (madeofpeople) [inline templates/html.html]: Its for windows 8

**Comment by madeofpeople** [comment]
I had issues with the boilerplate and drupal (http://actionswitchboard.net)... showing default drupal favicon, the more elaborate version i provided fixed the issue...

**Comment by RobLoach** [comment]
For Drupal, that should be fixed with http://drupal.org/project/favicon .... What markup should KalaStatic core provide though?

**Comment by madeofpeople** [comment]
Well i figure we should go for the most coverage.
I don't personally care about windows 8 but I'm sure it represents thousands of users.

We can merge this though, since its more than what we have, and we can always improve.

**Pull request merged**